src/Components/LogIn/LogIn.jsx

```diff
@@ -1,62 +1,59 @@
-import { useFormik } from "formik";
-import * as Yup from "yup";
-import "./login.css";
-
+import "./Login.css";
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
 const Login = () => {
-  const formik = useFormik({
-    initialValues: {
-      username: "",
-      password: "",
-    },
-    validationSchema: Yup.object({
-      // USER NAME
-      username: Yup.string()
-        .required("Please enter your username")
-        .min(4, "Must be 4 characters or more"),
-      //PASSWORD
-      password: Yup.string()
-        .required("Please enter your password")
-        .matches(
-          /^(?=.*[a-zA-Z])(?=.*\d)(?=.*[!@#$%^&*()_+])[A-Za-z\d][A-Za-z\d!@#$%^&*()_+]{8,14}$/,
-          "Password must be 8-14 characters and contain at least one letter, one number and a special character"
-        ),
-    }),
-    onSubmit: (values) => {
-      window.alert("Form submitted");
-      console.log(values);
-    },
-  });
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
 
+  const handleLogin = (e) => {
+    e.preventDefault();
+    const newUser = {
```

Comment:
just user 
newUser sounds like it's for a signup function

Reply:
e sửa lại rồi ạ. a approve giúp e với